crates/ursa-pod/benchmarks/terraform/main.tf

```diff
@@ -0,0 +1,38 @@
+# AWS Setup, requires `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
```

Comment:
Infra stuff can be moved to infra folder in root

Reply:
oops! this was intended for the next pr, still need to finish it

Reply:
decided to just include the terraform stuff with this PR anyway, https://github.com/fleek-network/ursa/pull/473/commits/fd3878e571ef3ab0d5ca5791a4b45a46772265eb

Reply:
ended up including it in this pr under `infra/ufdp-benchmark`